lib/fs.js

```diff
@@ -40,7 +40,6 @@ const isWindows = process.platform === 'win32';
 
 const DEBUG = process.env.NODE_DEBUG && /fs/.test(process.env.NODE_DEBUG);
 const errnoException = util._errnoException;
-const printDeprecation = require('internal/util').printDeprecationMessage;
```

Comment:
There are several other open PRs that move bits to `internal/fs.js` that would cover this requirement.